core-concepts/wallets/veworld/user-guide/wallet.md

```diff
@@ -0,0 +1,244 @@
+| <p>The first time you install the app you should:</p><ol><li>Tap the <strong>Get Started</strong> button</li><li>Follow the below steps</li><li>Protect your wallet by choosing your favorite method (FaceId, Fingerprint or password)</li></ol> | <p>If you already have a wallet on VeWorld, and you want to create a new one:</p><ol><li>From your dashboard tap the top-right wallet icon <img src="../../../../.gitbook/assets/veworld/account_balance_wallet.svg" alt=""></li><li>Tap the <strong>Add Wallet</strong> button</li><li>Follow the below steps</li></ol> |
+
+**Steps for a new wallet:**
+
+1. Tap the **Create new wallet** button
+2. Read and store your mnemonic
+3. Mark the checkbox that states you saved your mnemonic somewhere
+4. Pass the quiz with the 3 words from your mnemonic
+{% endtab %}
+
+{% tab title="Extension" %}
+| Fresh Start                                                                                                                                                                                                                                                      | Add a new one                                                                                                                                                                                                                                                                                                                                                |
+| ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| <p>The first time you install the extension you should:</p><ol><li>Follow and complete or skip the <em>welcome wizard</em></li><li>Type your favorite password twice and then click the <strong>CONFIRM</strong> button</li><li>Follow the below steps</li></ol> | <p>If you already have a wallet on VeWorld, and you want to create a new one:</p><ol><li>From your dashboard tap the top-right 3-dots icon <img src="../../../../.gitbook/assets/veworld/more_vert.svg" alt=""></li><li>Select <strong>Manage Wallets</strong></li><li>Click the <strong>Add Wallet</strong> button</li><li>Follow the below steps</li></ol> |
```

Comment:
I would keep the whole "browser extension" for clarity instead of "extension" only.